src/ml/neural_net/compute_context.hpp

```diff
@@ -12,6 +12,7 @@
 #include <core/export.hpp>
 #include <ml/neural_net/image_augmentation.hpp>
 #include <ml/neural_net/model_backend.hpp>
+#include <model_server/lib/variant.hpp>
```

Comment:
Why was this added?

Reply:
Removed. Added by mistake when validation_data was being passed in `create_drawing_classifier`.